catalog/spatial/csw/spatial-csw-source/pom.xml

```diff
@@ -172,6 +177,7 @@
                         <!-- Don't Export Anything -->
                         <Export-Package/>
                         <Embed-Dependency>
+                            spatial-csw-source-common,
                             spatial-csw-common,
                             spatial-ogc-common,
                             spatial-csw-transformer,
```

Comment:
I saw that and cringed myself. I think that changing this dependency could be part of the next tickets refactoring.